src/app/shared/users/user-list/user-list.component.ts

```diff
@@ -95,4 +95,13 @@ export class UserListComponent {
     const newPermission = {id: userId, roles: data.roles};
     this.userPermissionChange.emit({newPermission, currentPermission, onlyStore: data.onlyStore});
   }
+
+  public lightenColor(color: string): string {
+    return HtmlModifier.shadeColor(color, .5);
```

Comment:
We use this method in many classes. Maybe we should extract it to a reusable pipe with optional percentage parameter.